src/tsolvers/lasolver/CutCreator.cc

```diff
@@ -100,10 +100,10 @@ bool normalizeRow(ColMatrix & A, RowIndex rowIndex, ColIndex pivotIndex, ColMatr
             return A[first].getFirstCoeff() < A[second].getFirstCoeff();
         });
         std::iter_swap(it, activeColumns.begin());
-        // Now the index of column with smallest value is first in activeColumns
+        // Now the index of column with smallest first non-zero value is first in activeColumns
```

Comment:
Every column in `activeColumns` has non-zero value for the current row. This additional text is confusing for me.

Reply:
I can remove it... It's probably about what the reader has as implicit assumptions I suppose.  It helped me somehow.